Rawl_B502/de_genelogia_síl_ébir.trig

```diff
@@ -45,17 +45,21 @@
     a foaf:Person;
     irishRel:genName "Ítha" ;
     owl:sameAs <http://example.com/Rawl_B502/genelach_na_fomore.trig#Ítha>.
-    
+
 <#MugNuadat>
     a foaf:Person ;
     irishRel:nomName "Mug Nuadat" ;
-    owl:sameAs <http://example.com/Rawl_B502/mínigud_senchais_ébir_inso.trig#MugNuadat>.
+    owl:sameAs <http://example.com/Rawl_B502/mínigud_senchais_ébir_inso.trig#MugNuadat>;
+    owl:sameAs <http://example.com/Rawl_B502/genelach_éoganachta_caissil_16.trig#MogaNuadat>;
+    owl:sameAs <http://example.com/Laud_Misc_610/CGH/do_minigud_senchais_fer_muman.trig#EoganTáidlech>.
```

Comment:
Are we sure that Eogan Táidlech is the same?

Reply:
Good point. However, this section states that they are the same:

> Éogan Taídlech .i. Mog Nuadat .i. dalta do Nuadait m. Dárfhine m. Bairr, dia m-baí oc dénam a rátha is aire ro gaired de-seom Mog.